src/components/CodeAssignmentModal/index.jsx

```diff
@@ -285,6 +300,16 @@ export class BaseCodeAssignmentModal extends React.Component {
       };
     }
 
+    let usersResponse = [];
+    try {
+      const response = await LmsApiService.fetchUserDetailsFromEmail({ emails: options.emails });
+      usersResponse = response.data;
+    } catch (error) {
+      logError(error);
+      throw new SubmissionError({ _error: error });
+    }
+    options.users = this.getCleanedUsers(options.emails, usersResponse);
```

Comment:
@HammadAhmadWaqas Please verify that you are expecting values in `users` variable.

Reply:
yes looks good.